### PR TITLE
Run load tests on the server

### PR DESCRIPTION
**Pull request by panjan**

### CODE REVIEW

**Comment by gdgate** [comment]
Merge Failed.

This change or one of its cross-repo dependencies was unable to be automatically merged with the current state of its repository. Please rebase the change and upload a new patchset.

**Comment by gdgate** [comment]
Merge Failed.

This change or one of its cross-repo dependencies was unable to be automatically merged with the current state of its repository. Please rebase the change and upload a new patchset.

**Comment by gdgate** [comment]
Merge Failed.

This change or one of its cross-repo dependencies was unable to be automatically merged with the current state of its repository. Please rebase the change and upload a new patchset.

**Comment by gdgate** [comment]
Merge Failed.

This change or one of its cross-repo dependencies was unable to be automatically merged with the current state of its repository. Please rebase the change and upload a new patchset.

**Comment by gdgate** [comment]
Merge Failed.

This change or one of its cross-repo dependencies was unable to be automatically merged with the current state of its repository. Please rebase the change and upload a new patchset.

**Comment by gdgate** [comment]
Merge Failed.

This change or one of its cross-repo dependencies was unable to be automatically merged with the current state of its repository. Please rebase the change and upload a new patchset.

**Comment by gdgate** [comment]
Merge Failed.

This change or one of its cross-repo dependencies was unable to be automatically merged with the current state of its repository. Please rebase the change and upload a new patchset.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/1093/ : SUCCESS in 8s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-build-pipeline/339/ : SUCCESS in 7m 45s

**Comment by panjan** [comment]
Should be merged **together** with the clean-up part (TBD).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/1094/ : SUCCESS in 9s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-build-pipeline/340/ : SUCCESS in 8m 57s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/1095/ : SUCCESS in 8s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-build-pipeline/341/ : SUCCESS in 8m 49s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/1096/ : SUCCESS in 8s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-build-pipeline/342/ : SUCCESS in 7m 45s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/1101/ : SUCCESS in 10s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-build-pipeline/347/ : SUCCESS in 9m 19s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/1111/ : SUCCESS in 10s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-build-pipeline/354/ : SUCCESS in 8m 24s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/1112/ : SUCCESS in 12s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-build-pipeline/355/ : SUCCESS in 8m 53s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/1114/ : SUCCESS in 13s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-build-pipeline/357/ : SUCCESS in 9m 31s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/1115/ : SUCCESS in 10s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-build-pipeline/358/ : SUCCESS in 8m 42s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/1117/ : SUCCESS in 9s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-build-pipeline/360/ : SUCCESS in 9m 05s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/1120/ : SUCCESS in 31s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-build-pipeline/363/ : SUCCESS in 8m 24s

File: spec/lcm/integration/brick_runner.rb [hunk 1]
```diff
@@ -26,8 +28,42 @@ def rollout_brick(opts)
...
+    def schedule_brick(brick_name, service_project, opts)
+      opts[:template_path] = "../../integration/params/#{brick_name}.json.erb" unless opts[:template_path]
+      decoded_params = GoodData::Helpers.decode_params(script_params(opts))
+      params, hidden_params = extract_hidden_params(decoded_params)
+      params.delete('gd_encoded_hidden_params')
+      deploy_string = "${PRODUCTION_APPSTORE}:branch/lcm2:/apps/#{brick_name}"
+
+      process = service_project.deploy_process(
+        deploy_string,
+        name: "lcm load test #{brick_name}"
```
Comment (Hahihula) [inline spec/lcm/integration/brick_runner.rb]: we should make the name more generic ... it's generic method

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/1125/ : SUCCESS in 10s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-build-pipeline/368/ : SUCCESS in 7m 50s

File: spec/lcm/load/shared_contexts_for_load_tests.rb [hunk 2]
```diff
@@ -1,5 +1,6 @@
 shared_context 'load tests' do
   after(:all) do
-    puts `GD_ENV=performance bundle exec ruby bin/test_projects_cleanup.rb -d 0 -f`
+    system('GD_ENV=performance bundle exec ruby bin/test_projects_cleanup.rb -d 0 -f') ||
```
Comment (Hahihula) [inline spec/lcm/load/shared_contexts_for_load_tests.rb]: maybe the puts should stay there...

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/1126/ : SUCCESS in 9s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-build-pipeline/369/ : SUCCESS in 8m 53s

**Pull request merged**